src/metabase/models/field_values.clj

```diff
@@ -175,6 +175,12 @@
 ;;; |                                                  Utils fns                                                     |
 ;;; +----------------------------------------------------------------------------------------------------------------+
 
+(defn inactive?
+  "If FieldValues have not been accessed recently they are considered inactive."
+  [field-values]
+  (and field-values (t/before? (:last_used_at field-values)
```

Comment:
nit: should we move the `nil` check to usage and let this handle the time check only?

Reply:
I prefer callers not to have to worry about `nil` checks unless it's very performance intensive. I could see an argument to use `when` rather than `and` for proper nil-punning but a clear boolean can be nice too.